requirements/requirements.in

```diff
@@ -12,7 +12,7 @@ cryptography>=41.0.7  # CVE-2023-49083
 Cython<3 # due to https://github.com/yaml/pyyaml/pull/702
 daphne
 distro
-django==4.2.10  # CVE-2024-24680
+django==4.2.14  # CVE-2024-39329
```

Comment:
@thedoubl3j we should probably unpin this? like >= instead of ==?

Reply:
no, we do not unpin django.